beam/www/beam/components/ControlButtons.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="control-buttons">
+		<button @click="onCreate" :disabled="docstatus !== -1">Save</button>
```

Comment:
The `docstatus` checks will probably have to change to account for Frappe's enum values and conditions:

```mermaid
stateDiagram-v2
	state "Draft/Unsaved (0)" as draft
	state "Submitted (1)" as submitted
	state "Cancelled (2)" as cancelled

    [*] --> draft

    draft --> submitted: Submit
    submitted --> cancelled: Cancel
```

Reply:
Hey @Alchez! Thanks for you review.

I just updated these buttons. Could you tell me if it's ok that way?